src/modules/products/products.service.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpStatus, Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { CreateProductDto } from './dto/create-product.dto';
+import { Product } from './entities/product.entity';
+
+@Injectable()
+export class ProductsService {
+  constructor(
+    @InjectRepository(Product)
+    private productRepository: Repository<Product>
+  ) {}
+
+  async remove(id: string) {
```

Comment:
Wrap points of database calls in try/catch block with proper error logging in the catch. Reference auth service for example

Reply:
@Heba-WebDev said using try catch wasn't important

Reply:
You can have a try catch in your controller to log the error or follow @Cobidex, no problem